demo/run-osm-demo.sh

```diff
@@ -22,7 +22,7 @@ BOOKSTORE_NAMESPACE="${BOOKSTORE_NAMESPACE:-bookstore}"
 BOOKTHIEF_NAMESPACE="${BOOKTHIEF_NAMESPACE:-bookthief}"
 BOOKWAREHOUSE_NAMESPACE="${BOOKWAREHOUSE_NAMESPACE:-bookwarehouse}"
 CERT_MANAGER="${CERT_MANAGER:-tresor}"
-CTR_REGISTRY="${CTR_REGISTRY:-osmci.azurecr.io/osm}"
+CTR_REGISTRY="${CTR_REGISTRY:-localhost:5000}"
```

Comment:
We should update the default value in the comments for CTR_REGISTRY in `.env.example`. Doesn't need to be this PR.

Reply:
Looks like that's already the default. There's an ACR registry listed as an example, but `localhost:5000` is listed as the default.